docs/data/quick-start.md

```diff
@@ -0,0 +1,802 @@
+---
+title: Quick Start
+description: The quick start guide of how to use amplitude SDK
+---
+## 1. Initialize the library
+
+=== "Browser"
+
+    <h4> Install the Dependency </h4>
+    ```bash title="NPM"
+
+    npm install @amplitude/analytics-browser
+
+    ```
+
+    ```bash title="YARN"
+
+    yarn add @amplitude/analytics-browser
+    ```
+
+    ```html title="Script loader"
+
+    <script type="text/javascript">
+    !function(){"use strict";!function(e,t){var r=e.amplitude||{_q:[]};if(r.invoked)e.console&&console.error&&console.error("Amplitude snippet has been loaded.");else{r.invoked=!0;var n=t.createElement("script");n.type="text/javascript",n.integrity="sha384-XNX6U2ua04l5JNPk8racSkagg14UYkjWQjQmbRhYhLh0rtnEFZ1QTynnf4EUTukV",n.crossOrigin="anonymous",n.async=!0,n.src="https://cdn.amplitude.com/libs/analytics-browser-1.1.1-min.js.gz",n.onload=function(){e.amplitude.runQueuedFunctions||console.log("[Amplitude] Error: could not load SDK")};var s=t.getElementsByTagName("script")[0];function v(e,t){e.prototype[t]=function(){return this._q.push({name:t,args:Array.prototype.slice.call(arguments,0)}),this}}s.parentNode.insertBefore(n,s);for(var o=function(){return this._q=[],this},i=["add","append","clearAll","prepend","set","setOnce","unset","preInsert","postInsert","remove","getUserProperties"],a=0;a<i.length;a++)v(o,i[a]);r.Identify=o;for(var u=function(){return this._q=[],this},c=["getEventProperties","setProductId","setQuantity","setPrice","setRevenue","setRevenueType","setEventProperties"],l=0;l<c.length;l++)v(u,c[l]);r.Revenue=u;var p=["getDeviceId","setDeviceId","getSessionId","setSessionId","getUserId","setUserId","setOptOut","setTransport","reset"],d=["init","add","remove","track","logEvent","identify","groupIdentify","setGroup","revenue","flush"];function f(e){function t(t,r){e[t]=function(){var n={promise:new Promise((r=>{e._q.push({name:t,args:Array.prototype.slice.call(arguments,0),resolve:r})}))};if(r)return n}}for(var r=0;r<p.length;r++)t(p[r],!1);for(var n=0;n<d.length;n++)t(d[n],!0)}f(r),e.amplitude=r}}(window,document)}();
+    </script>
+    ```
+
+    <h4> Initialization </h4>
+
+    ```ts
+    import { init } from '@amplitude/analytics-browser';
+
+    init(AMPLITUDE_API_KEY, 'user@amplitude.com');
+    ```
+
+=== "Node"
+
+    <h4> Install the Dependency </h4>
+
+    ```bash title="NPM"
+
+    npm install @amplitude/analytics-node
+    ```
+
+    ```bash title="YARN"
+
+    yarn add @amplitude/analytics-node
+    ```
+
+    <h4> Initialization </h4>
+
+    ```ts
+    import { init } from '@amplitude/analytics-node';
+
+    init(AMPLITUDE_API_KEY);
+    ```
+
+=== "Android"
```

Comment:
nit: Can we have this slider default to browser? It's weird Browser is the first Tab but Android is selected by default.
<img width="724" alt="image" src="https://user-images.githubusercontent.com/5790872/189180879-e0b5607f-a4f6-4262-bf98-e71e27521cf8.png">

Reply:
I think it depends on which tab you visited last time. But if you are first time visiting this page, it will default to the first tab.